lib/ui/window/pointer_data_packet_converter.cc

```diff
@@ -261,8 +344,11 @@ PointerState PointerDataPacketConverter::EnsurePointerState(
   PointerState state;
   state.pointer_identifier = 0;
   state.is_down = false;
+  state.is_pan_zoom_active = false;
   state.physical_x = pointer_data.physical_x;
   state.physical_y = pointer_data.physical_y;
+  state.pan_x = pointer_data.pan_x;
```

Comment:
I understand that it makes logical sense, but in practice shouldn't pan start with (0, 0)? When will it not?

Reply:
You're right, It'll always start at (0,0). I will update it to put those values there, the current way is depending on the unset pan_x and pan_y being equal to 0 in some cases.